setup.cfg

```diff
@@ -27,6 +27,8 @@ install_requires =
     diffcalc-core
     fastapi
     uvicorn
+    pymongo
```

Comment:
Can remove the pymongo dependency now we have motor

Reply:
Ah, actually I use some of the types from pymongo because motor uses them also. So I've kept it in